python-base/ubuntu22.04-python3.11-nginx/config.yaml

```
@@ -0,0 +1,3 @@
+tags:
+  - ubuntu-python3.11-nginx
+  - ubuntu22-python3.11-nginx
```

Would prefer here as well to use ubuntu22.04 (as there's a 22.10 as well).
```suggestion
  - ubuntu22.04-python3.11-nginx
```